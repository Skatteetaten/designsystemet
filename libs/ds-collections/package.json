{
  "name": "@skatteetaten/ds-collections",
  "groupId": "no.skatteetaten.aurora",
  "version": "1.4.0",
  "license": "Apache-2.0",
  "homepage": "https://www.skatteetaten.no/stilogtone/designsystemet/",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Skatteetaten/designsystemet.git"
  },
  "publishConfig": {
    "registry": "https://nexus.sits.no/repository/npm-internal/"
  },
  "peerDependencies": {
    "i18next": "^23",
    "react": "^18",
    "react-i18next": "^14",
    "@skatteetaten/ds-core-utils": "1.3.0",
    "@skatteetaten/ds-icons": "1.3.0",
    "@skatteetaten/ds-buttons": "1.3.0",
    "@skatteetaten/ds-content": "1.3.0",
    "@skatteetaten/ds-typography": "1.3.0",
    "@skatteetaten/ds-progress": "1.3.0"
  },
  "scripts": {}
}
