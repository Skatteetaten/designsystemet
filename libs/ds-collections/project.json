{
  "name": "ds-collections",
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/ds-collections/src",
  "projectType": "library",
  "tags": [],
  "implicitDependencies": ["!ds-dev-config"],
  "targets": {
    "lint": {
      "executor": "@nx/eslint:lint",
      "outputs": ["{options.outputFile}"]
    },
    "build": {
      "executor": "@nx/rollup:rollup",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/ds-collections",
        "tsConfig": "libs/ds-collections/tsconfig.lib.json",
        "project": "libs/ds-collections/package.json",
        "entryFile": "libs/ds-collections/src/index.ts",
        "external": ["react/jsx-runtime"],
        "rollupConfig": ["libs/ds-collections/rollup.config.js"],
        "compiler": "swc",
        "assets": [
          {
            "glob": "libs/ds-collections/README.md",
            "input": ".",
            "output": "."
          },
          {
            "glob": "LICENSE",
            "input": ".",
            "output": "."
          }
        ],
        "babelUpwardRootMode": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/ds-collections/jest.config.ts"
      }
    },
    "stylelint": {
      "executor": "nx-stylelint:lint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": [
          "libs/ds-collections/**/*.css",
          "libs/ds-collections/**/*.scss"
        ]
      }
    },
    "tsc": {
      "executor": "@skatteetaten/ds-dev-config:tsCheck",
      "options": {
        "tsConfig": ["tsconfig.lib.json", "tsconfig.spec.json"]
      },
      "dependsOn": [
        {
          "projects": "ds-core-designtokens",
          "target": "build"
        }
      ]
    },
    "set-version": {
      "executor": "nx:run-commands",
      "options": {
        "command": "npm version {args.packageVersion} --no-git-tag",
        "cwd": "libs/ds-collections"
      }
    },
    "publish": {
      "executor": "nx:run-commands",
      "options": {
        "command": "npm publish dist/libs/ds-collections {args.switches}",
        "cwd": ""
      }
    }
  }
}
