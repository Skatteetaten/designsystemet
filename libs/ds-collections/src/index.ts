export * from './Accordion/Accordion';
export * from './Accordion/Accordion.types';
export * from './AccordionItem/AccordionItem.types';
export * from './Chips/Chips';
export * from './Chips/Chips.types';
export * from './ChipsToggle/ChipsToggle';
export * from './ChipsToggle/ChipsToggle.types';
export * from './ChipsRemovable/ChipsRemovable';
export * from './ChipsRemovable/ChipsRemovable.types';
export * from './OpenClose/OpenClose';
export * from './OpenClose/OpenClose.types';
export * from './StepList/StepList';
export * from './StepList/StepList.types';
export * from './StepListStep/StepListStep.types';
export * from './Tabs/Tabs';
export * from './Tabs/Tabs.types';
