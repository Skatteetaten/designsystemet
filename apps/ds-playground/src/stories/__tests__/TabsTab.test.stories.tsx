import { Tabs } from '@skatteetaten/ds-collections';
import {
  DeploySVGpath,
  LockSVGpath,
  PersonSVGpath,
} from '@skatteetaten/ds-icons';
import { Meta, StoryFn, StoryObj } from '@storybook/react';
import { expect, within } from '@storybook/test';

const meta = {
  component: Tabs.Tab,
  title: 'Tester/Tabs/Tab',
  argTypes: {
    // Baseprops
    key: { table: { disable: true } },
    ref: { table: { disable: true } },
    className: { table: { disable: true } },
    id: { table: { disable: true } },
    lang: { table: { disable: true } },
    'data-testid': { table: { disable: true } },
    // Props
    children: { table: { disable: true } },
    svgPath: { table: { disable: true } },
    value: { table: { disable: true } },
  },
} satisfies Meta<typeof Tabs.Tab>;
export default meta;
type Story = StoryObj<typeof meta>;

const defaultArgs = {
  children: 'Person',
};

const TemplateTabs: StoryFn<typeof Tabs.Tab> = (args) => {
  return (
    <Tabs defaultValue={'tab1'}>
      <Tabs.List>
        <Tabs.Tab {...args} value={'tab1'}>
          {'Person'}
        </Tabs.Tab>
        <Tabs.Tab {...args} value={'tab2'}>
          {'Bedrift'}
        </Tabs.Tab>
        <Tabs.Tab {...args} value={'tab3'}>
          {'Astronaut'}
        </Tabs.Tab>
      </Tabs.List>
      <Tabs.Panel value={'tab1'}>{'Tabs.Panel Person'}</Tabs.Panel>
      <Tabs.Panel value={'tab2'}>{'Tabs.Panel Bedrift'}</Tabs.Panel>
      <Tabs.Panel value={'tab3'}>{'Tabs.Panel Astronaut'}</Tabs.Panel>
    </Tabs>
  );
};

const TemplateTabsTab: StoryFn<typeof Tabs.Tab> = (args) => {
  return (
    <Tabs defaultValue={'tab1'}>
      <Tabs.List>
        <Tabs.Tab {...args}>{args.children}</Tabs.Tab>
        <Tabs.Tab value={'tab2'}>{'Bedrift'}</Tabs.Tab>
      </Tabs.List>
      <Tabs.Panel value={'tab1'}>{'Tabs.Panel Person'}</Tabs.Panel>
      <Tabs.Panel value={'tab2'}>{'Tabs.Panel Bedrift'}</Tabs.Panel>
    </Tabs>
  );
};

const TemplateTabsIcon: StoryFn<typeof Tabs.Tab> = (args) => {
  return (
    <>
      <Tabs defaultValue={'tab1'}>
        <Tabs.List>
          <Tabs.Tab {...args}>{args.children}</Tabs.Tab>
          <Tabs.Tab {...args} svgPath={LockSVGpath} value={'tab2'}>
            {'Bedrift'}
          </Tabs.Tab>
          <Tabs.Tab {...args} svgPath={DeploySVGpath} value={'tab3'}>
            {'Astronaut'}
          </Tabs.Tab>
        </Tabs.List>
        <Tabs.Panel value={'tab1'}>{'Tabs.Panel Person'}</Tabs.Panel>
        <Tabs.Panel value={'tab2'}>{'Tabs.Panel Bedrift'}</Tabs.Panel>
        <Tabs.Panel value={'tab3'}>{'Tabs.Panel Astronaut'}</Tabs.Panel>
      </Tabs>

      <Tabs defaultValue={'tab1'} variant={'compact'}>
        <Tabs.List>
          <Tabs.Tab {...args}>{'Person 2'}</Tabs.Tab>
          <Tabs.Tab {...args} svgPath={LockSVGpath} value={'tab2'}>
            {'Bedrift 2'}
          </Tabs.Tab>
          <Tabs.Tab {...args} svgPath={DeploySVGpath} value={'tab3'}>
            {'Astronaut 2'}
          </Tabs.Tab>
        </Tabs.List>
        <Tabs.Panel value={'tab1'}>
          {'Tabs.Panel Person 2, variant compact'}
        </Tabs.Panel>
        <Tabs.Panel value={'tab2'}>
          {'Tabs.Panel Bedrift 2, variant compact'}
        </Tabs.Panel>
        <Tabs.Panel value={'tab3'}>
          {'Tabs.Panel Astronaut 2, variant compact'}
        </Tabs.Panel>
      </Tabs>
    </>
  );
};

const TemplateTabsBorder: StoryFn<typeof Tabs.Tab> = (args) => {
  return (
    <Tabs defaultValue={'tab1'} hasBorder>
      <Tabs.List>
        <Tabs.Tab {...args} svgPath={PersonSVGpath} value={'tab1'}>
          {'Person'}
        </Tabs.Tab>
        <Tabs.Tab {...args} svgPath={LockSVGpath} value={'tab2'}>
          {'Bedrift'}
        </Tabs.Tab>
      </Tabs.List>
      <Tabs.Panel value={'tab1'}>{'Tabs.Panel Person'}</Tabs.Panel>
      <Tabs.Panel value={'tab2'}>{'Tabs.Panel Bedrift'}</Tabs.Panel>
    </Tabs>
  );
};

export const Defaults = {
  name: 'Defaults (A2)',
  render: TemplateTabsTab,
  args: {
    children: 'TabText',
    value: 'tab1',
  },
  argTypes: {
    svgPath: { table: { disable: false } },
  },
  play: async ({ canvasElement }): Promise<void> => {
    const canvas = within(canvasElement);
    const tab = canvas.getByRole('tab', { name: 'TabText' });
    await expect(tab).toBeInTheDocument();
  },
} satisfies Story;

export const WithAttributes = {
  name: 'With Attributes (FA2-5, B1)',
  render: TemplateTabs,
  args: {
    ...defaultArgs,
    className: 'dummyClassname',
    lang: 'nb',
    'data-testid': '123ID',
    value: 'TabValue',
  },
  argTypes: {
    id: { table: { disable: true } },
    className: { table: { disable: false } },
    lang: { table: { disable: false } },
    'data-testid': { table: { disable: false } },
  },
  parameters: {
    HTMLSnapshot: { disable: true },
    imageSnapshot: { disable: false },
  },
  play: async ({ canvasElement, step }): Promise<void> => {
    const canvas = within(canvasElement);
    const tab = canvas.getByRole('tab', { name: 'Bedrift' });
    await step(
      'Autogenerert id-attributt basert pÃ¥ tab name "tab2"',
      async () => {
        await expect(tab).toHaveAttribute(
          'id',
          expect.stringMatching(/^ds-tab-id-.*-tab2$/)
        );
      }
    );
    await expect(tab).toHaveClass('dummyClassname');
    await expect(tab).toHaveAttribute('lang', 'nb');
    await expect(tab).toHaveAttribute('data-testid', '123ID');
  },
} satisfies Story;

export const WithIcon = {
  name: 'With Icon (A4)',
  render: TemplateTabsIcon,
  args: {
    ...defaultArgs,
    svgPath: PersonSVGpath,
    value: 'tab1',
  },
  parameters: {
    imageSnapshot: { disable: false },
  },
  play: async ({ canvasElement, step }): Promise<void> => {
    const canvas = within(canvasElement);
    await step('Sjekk om svg-ikon finnes', async () => {
      const tab = canvas.getByRole('tab', { name: 'Person' });
      const svg = tab.querySelector('svg');
      await expect(svg).toBeInTheDocument();
    });
  },
} satisfies Story;

export const WithBorder = {
  name: 'With Border (A1',
  render: TemplateTabsBorder,
  args: {
    ...defaultArgs,
    value: 'tab2',
  },
} satisfies Story;
