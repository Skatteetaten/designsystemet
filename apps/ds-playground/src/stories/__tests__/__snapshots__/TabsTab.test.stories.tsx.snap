// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tester/Tabs/Tab Defaults play-test 1`] = `
<div data-test-block="true">
  <div class>
    <div class="_tabList_ibyzk_1"
         role="tablist"
         tabindex="-1"
    >
      <button id="ds-tab-id-:r0:-tab1"
              class="_tab_1dx22_1   _tab_active_1dx22_27"
              role="tab"
              tabindex="0"
              aria-selected="true"
              aria-controls="ds-tab-panel-:r0:-tab1"
      >
        TabText
      </button>
      <button id="ds-tab-id-:r0:-tab2"
              class="_tab_1dx22_1"
              role="tab"
              tabindex="-1"
              aria-selected="false"
              aria-controls="ds-tab-panel-:r0:-tab2"
      >
        Bedrift
      </button>
    </div>
    <div id="ds-tab-panel-:r0:-tab1"
         class="_panel_1bj6c_1 _panel_active_1bj6c_4"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r0:-tab1"
    >
      Tabs.Panel Person
    </div>
    <div id="ds-tab-panel-:r0:-tab2"
         class="_panel_1bj6c_1"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r0:-tab2"
         hidden
    >
      Tabs.Panel Bedrift
    </div>
  </div>
</div>
`;

exports[`Tester/Tabs/Tab WithBorder smoke-test 1`] = `
<div data-test-block="true">
  <div class>
    <div class="_tabList_ibyzk_1"
         role="tablist"
         tabindex="-1"
    >
      <button id="ds-tab-id-:ra:-tab1"
              class="_tab_1dx22_1  _tab_border_1dx22_51 _tab_active_1dx22_27 _tab_icon_1dx22_59"
              role="tab"
              tabindex="0"
              aria-selected="true"
              aria-controls="ds-tab-panel-:ra:-tab1"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M12 4a4 4 0 1 1 0 8 4 4 0 0 1 0-8Zm0 10c4.42 0 8 1.79 8 4v2H4v-2c0-2.21 3.58-4 8-4Z">
          </path>
        </svg>
        Person
      </button>
      <button id="ds-tab-id-:ra:-tab2"
              class="_tab_1dx22_1  _tab_border_1dx22_51  _tab_icon_1dx22_59"
              role="tab"
              tabindex="-1"
              aria-selected="false"
              aria-controls="ds-tab-panel-:ra:-tab2"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M12 17a2 2 0 1 0 0-4 2 2 0 0 0 0 4Zm6-9a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V10a2 2 0 0 1 2-2h1V6a5 5 0 1 1 10 0v2h1Zm-6-5a3 3 0 0 0-3 3v2h6V6a3 3 0 0 0-3-3Z">
          </path>
        </svg>
        Bedrift
      </button>
    </div>
    <div id="ds-tab-panel-:ra:-tab1"
         class="_panel_1bj6c_1 _panel_active_1bj6c_4"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:ra:-tab1"
    >
      Tabs.Panel Person
    </div>
    <div id="ds-tab-panel-:ra:-tab2"
         class="_panel_1bj6c_1"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:ra:-tab2"
         hidden
    >
      Tabs.Panel Bedrift
    </div>
  </div>
</div>
`;

exports[`Tester/Tabs/Tab WithIcon play-test 1`] = `
<div data-test-block="true">
  <div class>
    <div class="_tabList_ibyzk_1"
         role="tablist"
         tabindex="-1"
    >
      <button id="ds-tab-id-:r2:-tab1"
              class="_tab_1dx22_1   _tab_active_1dx22_27 _tab_icon_1dx22_59"
              role="tab"
              tabindex="0"
              aria-selected="true"
              aria-controls="ds-tab-panel-:r2:-tab1"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M12 4a4 4 0 1 1 0 8 4 4 0 0 1 0-8Zm0 10c4.42 0 8 1.79 8 4v2H4v-2c0-2.21 3.58-4 8-4Z">
          </path>
        </svg>
        Person
      </button>
      <button id="ds-tab-id-:r2:-tab2"
              class="_tab_1dx22_1    _tab_icon_1dx22_59"
              role="tab"
              tabindex="-1"
              aria-selected="false"
              aria-controls="ds-tab-panel-:r2:-tab2"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M12 17a2 2 0 1 0 0-4 2 2 0 0 0 0 4Zm6-9a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V10a2 2 0 0 1 2-2h1V6a5 5 0 1 1 10 0v2h1Zm-6-5a3 3 0 0 0-3 3v2h6V6a3 3 0 0 0-3-3Z">
          </path>
        </svg>
        Bedrift
      </button>
      <button id="ds-tab-id-:r2:-tab3"
              class="_tab_1dx22_1    _tab_icon_1dx22_59"
              role="tab"
              tabindex="-1"
              aria-selected="false"
              aria-controls="ds-tab-panel-:r2:-tab3"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="m2.19 19.169 4.09-4.08c-.21-.3-.36-.62-.45-.97l-3.64 3.64v1.41Zm0 2.83H3.6l4.77-4.76-1.42-1.41-4.76 4.76v1.41Zm4.24 0 3.64-3.64c-.34-.09-.67-.24-.97-.45l-4.08 4.09h1.41Zm2.64-5.47-1.41-1.41 1.41 1.41Zm5.66-7.07c-.78-.78-.78-2.05 0-2.83.78-.78 2.05-.78 2.83 0 .77.78.78 2.05 0 2.83-.78.78-2.05.78-2.83 0Zm7.07-7.07S16.85.268 11.19 5.929c-2.19 2.19-3.5 4.6-4.35 6.71-.28.75-.09 1.57.46 2.13l2.13 2.12c.55.56 1.37.74 2.12.46a19.1 19.1 0 0 0 6.71-4.35c5.66-5.66 3.54-10.61 3.54-10.61ZM5.83 12.499 2 10.869l6.1-2.77c-.91 1.36-1.69 2.83-2.27 4.4Zm7.49 9.69-1.63-3.83c1.57-.58 3.04-1.36 4.4-2.27l-2.77 6.1Z">
          </path>
        </svg>
        Astronaut
      </button>
    </div>
    <div id="ds-tab-panel-:r2:-tab1"
         class="_panel_1bj6c_1 _panel_active_1bj6c_4"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r2:-tab1"
    >
      Tabs.Panel Person
    </div>
    <div id="ds-tab-panel-:r2:-tab2"
         class="_panel_1bj6c_1"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r2:-tab2"
         hidden
    >
      Tabs.Panel Bedrift
    </div>
    <div id="ds-tab-panel-:r2:-tab3"
         class="_panel_1bj6c_1"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r2:-tab3"
         hidden
    >
      Tabs.Panel Astronaut
    </div>
  </div>
  <div class>
    <div class="_tabList_ibyzk_1"
         role="tablist"
         tabindex="-1"
    >
      <button id="ds-tab-id-:r6:-tab1"
              class="_tab_1dx22_1 _tab_compact_1dx22_43  _tab_active_1dx22_27 _tab_icon_1dx22_59"
              role="tab"
              tabindex="0"
              aria-selected="true"
              aria-controls="ds-tab-panel-:r6:-tab1"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M12 4a4 4 0 1 1 0 8 4 4 0 0 1 0-8Zm0 10c4.42 0 8 1.79 8 4v2H4v-2c0-2.21 3.58-4 8-4Z">
          </path>
        </svg>
        Person 2
      </button>
      <button id="ds-tab-id-:r6:-tab2"
              class="_tab_1dx22_1 _tab_compact_1dx22_43   _tab_icon_1dx22_59"
              role="tab"
              tabindex="-1"
              aria-selected="false"
              aria-controls="ds-tab-panel-:r6:-tab2"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M12 17a2 2 0 1 0 0-4 2 2 0 0 0 0 4Zm6-9a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2H6a2 2 0 0 1-2-2V10a2 2 0 0 1 2-2h1V6a5 5 0 1 1 10 0v2h1Zm-6-5a3 3 0 0 0-3 3v2h6V6a3 3 0 0 0-3-3Z">
          </path>
        </svg>
        Bedrift 2
      </button>
      <button id="ds-tab-id-:r6:-tab3"
              class="_tab_1dx22_1 _tab_compact_1dx22_43   _tab_icon_1dx22_59"
              role="tab"
              tabindex="-1"
              aria-selected="false"
              aria-controls="ds-tab-panel-:r6:-tab3"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_small_dry7n_5"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="m2.19 19.169 4.09-4.08c-.21-.3-.36-.62-.45-.97l-3.64 3.64v1.41Zm0 2.83H3.6l4.77-4.76-1.42-1.41-4.76 4.76v1.41Zm4.24 0 3.64-3.64c-.34-.09-.67-.24-.97-.45l-4.08 4.09h1.41Zm2.64-5.47-1.41-1.41 1.41 1.41Zm5.66-7.07c-.78-.78-.78-2.05 0-2.83.78-.78 2.05-.78 2.83 0 .77.78.78 2.05 0 2.83-.78.78-2.05.78-2.83 0Zm7.07-7.07S16.85.268 11.19 5.929c-2.19 2.19-3.5 4.6-4.35 6.71-.28.75-.09 1.57.46 2.13l2.13 2.12c.55.56 1.37.74 2.12.46a19.1 19.1 0 0 0 6.71-4.35c5.66-5.66 3.54-10.61 3.54-10.61ZM5.83 12.499 2 10.869l6.1-2.77c-.91 1.36-1.69 2.83-2.27 4.4Zm7.49 9.69-1.63-3.83c1.57-.58 3.04-1.36 4.4-2.27l-2.77 6.1Z">
          </path>
        </svg>
        Astronaut 2
      </button>
    </div>
    <div id="ds-tab-panel-:r6:-tab1"
         class="_panel_1bj6c_1 _panel_active_1bj6c_4"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r6:-tab1"
    >
      Tabs.Panel Person 2, variant compact
    </div>
    <div id="ds-tab-panel-:r6:-tab2"
         class="_panel_1bj6c_1"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r6:-tab2"
         hidden
    >
      Tabs.Panel Bedrift 2, variant compact
    </div>
    <div id="ds-tab-panel-:r6:-tab3"
         class="_panel_1bj6c_1"
         role="tabpanel"
         aria-labelledby="ds-tab-id-:r6:-tab3"
         hidden
    >
      Tabs.Panel Astronaut 2, variant compact
    </div>
  </div>
</div>
`;
